backend/server/sponsor.go

```diff
@@ -42,15 +45,23 @@ func SponsorSetup(client *mongo.Client) {
 	}
 
 	// Fetching sponsor list
-	resp, err := http.Get("https://gistcdn.githack.com/esyw/4e35cd5fe73fa024020e67855ca733fb/raw/e85c9ae58a6323a4214ffa4ad89b0a5ebe404e31/sponsors.json")
-	if err != nil {
-		log.Fatal("Could not get sponsor list: ", err)
-	}
-	defer resp.Body.Close()
+	/*
+		resp, err := http.Get("https://gistcdn.githack.com/esyw/4e35cd5fe73fa024020e67855ca733fb/raw/e85c9ae58a6323a4214ffa4ad89b0a5ebe404e31/sponsors.json")
```

Comment:
Lets remove the commented out code if its not used anymore

Reply:
Yep we forgot about this.